cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,491 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-8180` - Fixed an issue with the CDAP Standalone not correctly warning
+  about the absence of Node.js.
+
+- :cask-issue:`CDAP-8229` - Fix the CDAP UpgradeTool to not rely on the existence of a
+  'default' namespace.
+
+- :cask-issue:`CDAP-8313` - Fixed an issue where system artifacts would continuously be
+  loaded if there was a partial JAR in the system artifacts directory.
+
+- :cask-issue:`CDAP-8342` - Fixed an issue where CDAP Explore operations from a program
+  container running as a user were impersonating the namespace owner. Now they impersonate
+  the respective program container users.
+
+- :cask-issue:`CDAP-8367` - Fixed an issue with "Hive-on-Spark" on newer versions of CDH.
+
+- :cask-issue:`CDAP-8442` - Fixed an issue in the CDAP UI where the "Stop Program" modal
+  dialog kept loading (showing a spinning wheel) even after the program had been stopped.
+
+- :cask-issue:`CDAP-8445` - The stream endpoint to enqueue messages now returns a 503
+  instead of a 500 if it failed because the dataset service was unavailable.
+
+- :cask-issue:`CDAP-8446` - Fixed an issue where the Transactional.run method could throw
+  the wrong exception if the transaction service was unavailable when it was finishing a
+  transaction.
+
+- :cask-issue:`CDAP-8448` - In general, fixed the HTTP RESTful endpoints to return a 503
+  instead of 500 when the transaction service was unavailable.
+
+- :cask-issue:`CDAP-8509` - Fixed an issue in the Transactional Messaging System (TMS)
+  table upgrade, where the TMS table could be left in a disabled state if the upgrade tool
+  is run after an upgraded CDAP Master is started and then stopped.
+
+- :cask-issue:`CDAP-8544` - Lowered the RPC timeout and number of retries for the HBase
+  operations performed by CDAP Master services.
+
+- :cask-issue:`CDAP-8628` - Fixed an issue in the log saver and the metrics processor that
+  if an exception was thrown during the changing of the number of instances, a container JVM
+  process could be left running without performing any work.
+
+- :cask-issue:`CDAP-8634` - Corrected the Javadoc of the PluginConfig's containsMacro()
+  method to reflect that it always returns false at runtime.
+
+- :cask-issue:`CDAP-8636` - Fixed an issue with Spark programs not working against CDH
+  5.8.4.
+
+- :cask-issue:`CDAP-8672` - Fixed the CDAP Router so that it does not log an error when it
+  cannot discover a service. Previously, the message was logged at the debug level.
+
+- :cask-issue:`CDAP-8687` - Fixed an issue where a user who attempts to create an existing
+  stream that was created by a different user received all the privileges and the original
+  user had their privileges revoked.
+
+- :cask-issue:`CDAP-8694` - Fixed an issue with properly-locating CDAP_HOME in Distributed
+  CDAP instances outside the default ``/opt/cdap`` directory.
+
+- :cask-issue:`CDAP-8753` - Deprecated the ``waitForFinish()`` method in the ProgramManager.
```

Comment:
Let's move to a "Deprecations" section? Should also mention what users should use instead